### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by FireArrow108**

### CODE REVIEW

File: MineAdminWF/Main.Designer.vb [hunk 1]
```diff
@@ -849,32 +867,45 @@ Partial Class Form
...
+        Me.bukkitFile.Anchor = System.Windows.Forms.AnchorStyles.None
+        Me.bukkitFile.Font = New System.Drawing.Font("Microsoft Sans Serif", 8.25!, System.Drawing.FontStyle.Italic, System.Drawing.GraphicsUnit.Point, CType(238, Byte))
+        Me.bukkitFile.Location = New System.Drawing.Point(3, 3)
+        Me.bukkitFile.Name = "bukkitFile"
+        Me.bukkitFile.Size = New System.Drawing.Size(174, 23)
+        Me.bukkitFile.TabIndex = 1
+        Me.bukkitFile.Text = "bukkit.yml"
+        Me.Tooltip.SetToolTip(Me.bukkitFile, "Opens bukkit.yml" & Global.Microsoft.VisualBasic.ChrW(13) & Global.Microsoft.VisualBasic.ChrW(10) & Global.Microsoft.VisualBasic.ChrW(13) & Global.Microsoft.VisualBasic.ChrW(10) & "This file contains configuration related to Bukkit servers (a" &
+        "nd possibly its forks)")
+        Me.bukkitFile.UseVisualStyleBackColor = True
```
Comment (FireArrow108) [inline MineAdminWF/Main.Designer.vb:908]: Fix button anchors

File: MineAdminWF/Main.Designer.vb [hunk 2]
```diff
@@ -889,17 +920,32 @@ Partial Class Form
...
+        Me.permissionsFile.Anchor = System.Windows.Forms.AnchorStyles.None
+        Me.permissionsFile.Font = New System.Drawing.Font("Microsoft Sans Serif", 8.25!, System.Drawing.FontStyle.Italic, System.Drawing.GraphicsUnit.Point, CType(238, Byte))
+        Me.permissionsFile.Location = New System.Drawing.Point(356, 26)
+        Me.permissionsFile.Name = "permissionsFile"
+        Me.permissionsFile.Size = New System.Drawing.Size(174, 23)
+        Me.permissionsFile.TabIndex = 6
+        Me.permissionsFile.Text = "permissions.yml"
+        Me.Tooltip.SetToolTip(Me.permissionsFile, "Opens premissions.yml" & Global.Microsoft.VisualBasic.ChrW(13) & Global.Microsoft.VisualBasic.ChrW(10) & Global.Microsoft.VisualBasic.ChrW(13) & Global.Microsoft.VisualBasic.ChrW(10) & "This file contains configuration for permissions the def" &
+        "ault group")
+        Me.permissionsFile.UseVisualStyleBackColor = True
```
Comment (FireArrow108) [inline MineAdminWF/Main.Designer.vb:948]: Fix button anchors

File: MineAdminWF/Main.Designer.vb [hunk 3]
```diff
@@ -914,31 +960,17 @@ Partial Class Form
...
-        Me.serverPropertiesFile.UseVisualStyleBackColor = True
+        Me.paperFile.Anchor = System.Windows.Forms.AnchorStyles.None
+        Me.paperFile.Font = New System.Drawing.Font("Microsoft Sans Serif", 8.25!, System.Drawing.FontStyle.Italic, System.Drawing.GraphicsUnit.Point, CType(238, Byte))
+        Me.paperFile.Location = New System.Drawing.Point(356, 3)
+        Me.paperFile.Name = "paperFile"
+        Me.paperFile.Size = New System.Drawing.Size(174, 23)
+        Me.paperFile.TabIndex = 3
+        Me.paperFile.Text = "paper.yml"
+        Me.Tooltip.SetToolTip(Me.paperFile, "Opens paper.yml" & Global.Microsoft.VisualBasic.ChrW(13) & Global.Microsoft.VisualBasic.ChrW(10) & Global.Microsoft.VisualBasic.ChrW(13) & Global.Microsoft.VisualBasic.ChrW(10) & "This file contains configuration related to Paper servers")
+        Me.paperFile.UseVisualStyleBackColor = True
```
Comment (FireArrow108) [inline MineAdminWF/Main.Designer.vb:973]: Fix button anchors

File: MineAdminWF/Main.vb [hunk 4]
```diff
@@ -210,6 +210,7 @@ Currently supporting 1.12.2, 1.14.4, 1.15.2 and 1.16.3", "Error", MessageBoxButt
                 ModsList.Items.AddRange(Mlist) 'Add mods to list
             Else
                 ModsNO.Visible = True
+                ModsList.Visible = False
```
Comment (FireArrow108) [inline MineAdminWF/Main.vb:213]: Make mod list invisible when it isn't compatible

**Review by FireArrow108 — COMMENTED** [review]
Run Code Cleanup (with sorting on)

**Review by Verarr — APPROVED** [review]
OK

**Pull request merged**